app/build.gradle

```diff
@@ -215,7 +215,8 @@ dependencies {
 
     androidTestImplementation "org.jetbrains.kotlin:kotlin-reflect:${kotlin_version}"
 
-    // See https://github.com/mozilla/application-services/blob/master/.buildconfig-android.yml#L1 for latest version
+    // See https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Dependencies.kt#L29
```

Comment:
This line is going to change so can we provide a link to the file and a constant name instead? e.g.:
```kotlin
// For the latest version, see `Versions.mozilla_appservices` in
// <link>